src/utils/isReactElement.ts

```diff
@@ -0,0 +1,5 @@
+import { ReactNode, FC, ReactElement } from 'react'
```

Comment:
Can you use the builtin `React.isValidElement(element)`?

Reply:
Awesome to know!